lighthouse/src/main.rs

```diff
@@ -756,6 +745,14 @@ fn run<E: EthSpec>(
     }
 
     match LighthouseSubcommands::from_arg_matches(matches) {
+        Ok(LighthouseSubcommands::AccountManager(account_manager_config)) => {
+            eprintln!("Running account manager for {} network", network_name);
+            // Pass the entire `environment` to the account manager so it can run blocking operations.
+            account_manager::run(&account_manager_config, matches, environment)?;
```

Comment:
We should reorder this for consistency with the other subcommands
```suggestion
            account_manager::run(matches, &account_manager_config, environment)?;
```

Reply:
[c0ce1cc](https://github.com/sigp/lighthouse/pull/6493/commits/c0ce1cceeab464925ea9d7a5cc4e2316fe35f186)